### PR TITLE
Added info how to create new internal api endpoints

### PR DESCRIPTION
**Pull request by abarkouski-sc**

### CODE REVIEW

File: Common/Applications/creating new internal api endpoints.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+   
+   /*
+   2) deny access to single controller myControllerApi only
+   
+   static excludes = ["/myControllerApi/**", "/MyControllerApi/**", "/myControllerApi", "/MyControllerApi"]
+   */
+   
+   // create mapping starting with /api:
+   static mappings = {
+       "/api/dosomething"(controller: 'myController', action: 'myMethod', method: "POST")
```
Comment (apechenko-sc) [inline Common/Applications/creating new internal api endpoints.md:46]: Is it `dosomething` or `something` (verb or noun)?

File: Common/Applications/creating new internal api endpoints.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+       def myMethod()) {
+       }
+   }
+   ```
+
+   1.2 We do not need Secured Resources for internal api calls. Secured Resource should not be generated for this controller.
+
+   Please, add controller name to an exclusion list.
+
+   for OPC: see `plugin.jcatalogAcegiSecurity.securedResource.excludedPaths`
```
Comment (apechenko-sc) [inline Common/Applications/creating new internal api endpoints.md:18]: It's probably in `Config.groovy`, but please add this info explicitly.

File: Common/Applications/creating new internal api endpoints.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+   
+   // create mapping starting with /api:
+   static mappings = {
+       "/api/dosomething"(controller: 'myController', action: 'myMethod', method: "POST")
+   }
+   ```
+
+3. Allow access to new endpoint for internal api calls, deny access from browser.
+
+   To do it: apply `apiSecurityFilterChain` for the endpoint. Means: add one line into `applicationContext-acegi-security.xml` :
```
Comment (apechenko-sc) [inline Common/Applications/creating new internal api endpoints.md:52]: Please, add a short explanation what `apiSecurityFilterChain` is doing?

File: Common/Applications/creating new internal api endpoints.md (hunk 2 above)
Comment (ivouchak-sc) [reply Common/Applications/creating new internal api endpoints.md]: Why in Config.groovy? It can be placed also in `SecuredResourcesConfiguration.groovy`

Comment (abarkouski-sc) [reply Common/Applications/creating new internal api endpoints.md]: Yes. I know. All ways to create/exclude Secured Resources - is a topic of a separate document, which needs to be created.

**Pull request merged**